docs/containerd-env-setup.md

```diff
@@ -6,6 +6,8 @@ This document will walk through how to setup a nydus image service to work with
 
 Get `nydus-image`, `nydusd`, `nydusify`, `ctr-remote` and `nydus-overlayfs` binaries from [image-service release](https://github.com/dragonflyoss/image-service/releases/latest) page.
 
+`nydusd` currently has two binaries, `nydusd-fusedev` (for fuse daemon mode) and `nydusd-virtiofs` (for virtiofs daemon mode), chose the one that fits your mode and rename it `nydusd`.
```

Comment:
We have a typo here? Should it the correct one be "choose"?
And "rename it as `nydusd` "would be better?